install.sh

```diff
@@ -284,11 +284,10 @@ if [ -z "$1" ] || contains "$*" vzlogger; then
 		echo "make sure to restart vzlogger"
 	fi
 
-	if ! grep -q vzlogger /etc/inittab; then
+	if [ ! -e "/etc/systemd/system/vzlogger.service" ]; then
 		echo
-		echo "could not find vzlogger in /etc/inittab"
-		echo "if you want vzlogger to start automatically add the following line to /etc/inittab"
-		echo "vz:235:respawn:/usr/local/bin/vzlogger -d"
-		echo "activate using `init q`"
+		echo "could not find vzlogger.service in /etc/systemd/system/"
+		echo "we recommend to configure vzlogger as a service"
+        echo "for further details: man systemd or wiki.volkszaehler.org"
```

Comment:
Der Code ist in Bezug auf Tabs furchtbar inkonsistent!
Ich hab meinen Editor extra umgestellt das es 4 blanks einfügt weil ich da immer wieder drüber gestolpert bin. Und es passt doch wieder nur in der Hälfte der Fälle. :( 
Fällt aber immer erst hinterher auf weil github die Tabs dann anders interpretiert.